pkg-scripts/postinstall.sh

```diff
@@ -1,3 +1,7 @@
 #!/bin/sh
+
+# WARN: path to config is referenced by plesk extension
+#       when enabling/disabling plugin thus plesk extension
```

Comment:
`thus` -> `this`

I don't understand the warning.

Reply:
PR is not necessary anymore